.circleci/config.yml

```diff
@@ -262,6 +262,34 @@ jobs:
       - report-workflow-on-failure
       - store_test_results:
           path: test-results
+  angular-prerelease-sandbox:
+    executor:
+      class: large
+      name: sb_playwright
+    steps:
+      - git-shallow-clone/checkout_advanced:
+          clone_options: '--depth 1 --verbose'
+      - attach_workspace:
+          at: .
+      - run:
+          name: Angular Prerelease Sandbox
+          command: |
+            node -v
+            yarn task --template angular-cli/prerelease --no-link --start-from=never --junit --task sandbox
+            yarn task --template angular-cli/prerelease --no-link --start-from=never --junit --task build
+            yarn task --template angular-cli/prerelease --no-link --start-from=never --junit --task chromatic
+            yarn task --template angular-cli/prerelease --no-link --start-from=never --junit --task e2e-tests
+            yarn task --template angular-cli/prerelease --no-link --start-from=never --junit --task e2e-tests-dev
+            yarn task --template angular-cli/prerelease --no-link --start-from=never --junit --task test-runner
```

Comment:
could you separate them into different commands so it's easier to know what failed if they do fail?